### PR TITLE
www.womenshealth.de

### PR DESCRIPTION
**Pull request by jellizaveta**
https://github.com/AdguardTeam/AdguardFilters/issues/118244

<details><summary>Screenshot:</summary>

Снимок экрана 2022-05-12 в 16 41 14

</details><br/>

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -3873,7 +3873,8 @@ financial.nikkei.com##.f-privacy
 nikkeibp.co.jp,nikkei.com##.bpPrivacy
 getmailbird.com##.navbar-cookie
 hashicorp.com###g-consent-manager
-spiegel.de,finn.no,vg.no,e24.no,aftenposten.no,tek.no,vglive.no,bt.no,telegraph.co.uk,gizmodo.com##div[id^="sp_message_container_"]
+womenshealth.de#%#//scriptlet('remove-class', 'sp-message-open', 'html')
+womenshealth.de,spiegel.de,finn.no,vg.no,e24.no,aftenposten.no,tek.no,vglive.no,bt.no,telegraph.co.uk,gizmodo.com##div[id^="sp_message_container_"]
```
[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: @BlazDT @AdamWr have you found any articles that require a JS rule? As I can see, the rule above works fine.

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (AdamWr) [reply AnnoyancesFilter/sections/cookies_specific.txt]: If I'm not wrong, video player doesn't work here - `https://www.womenshealth.de/fitness/laufen/joggen-anfangen/` if cookie notice is blocked.

[the remark this answers is not in the document]
Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt]: On my end player works with these rules:
`womenshealth.de#%#AG_onLoad(function(){if(window.location.href.includes("consent")){var g=new MutationObserver(function(){var b=document.querySelector('button[title^="Accept and continue"]');if(!b){var b=document.querySelector('button[title^="Akzeptieren und weiter"]');};b&&(g.disconnect(),b.click())});g.observe(document,{childList:!0,subtree:!0});setTimeout(function(){g.disconnect()},1E4)}});`
with unblocking/redirect for German filter
```
@@||cdn.stroeerdigitalgroup.de/metatag/live/mps_womenshealth/metaTag.min.js$domain=womenshealth.de
||cdn.stroeerdigitalgroup.de/metatag/live/mps_womenshealth/metaTag.min.js$script,redirect=noopjs,domain=womenshealth.de,important
```

[the remark this answers is not in the document]
Comment (zloyden) [reply AnnoyancesFilter/sections/cookies_specific.txt]: @Alex-302 they work fine on my end.

[the remark this answers is not in the document]
Comment (AdamWr) [reply AnnoyancesFilter/sections/cookies_specific.txt]: These rules work fine on my end too, but I guess that JS rule can be added just for `privacy.womenshealth.de` and it also should works fine.

[the remark this answers is not in the document]
Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_specific.txt]: @Alex-302 `womenshealth.de` for Safari. If `privacy.womenshealth.de` - the rule won't work.

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**